sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -242,6 +246,19 @@ private static void validateFormLineData(List<TextLine> expectedLines, List<Form
         }
     }
 
+    private static void validateFormSelectionMarkData(List<SelectionMark> expectedMarks,
+        List<FormSelectionMark> actualMarks, int pageNumber) {
+        for (int i = 0; i < actualMarks.size(); i++) {
+            final SelectionMark expectedMark = expectedMarks.get(i);
+            final FormSelectionMark actualMark = actualMarks.get(i);
+            assertTrue(expectedMark.getState().toString().equalsIgnoreCase(actualMark.getState().toString()));
```

Comment:
Why do we need to ignore the case, we should have same casing.

Reply:
The reason of using `equalIgnoreCae` method is because Service level, SelectionMarkState, expectedMark's  mark state is using ExpandableEnum which has `selected` but in SDK level, it is an enum, which uses `SELECTED` as the value. 

Since we are trying to using expandable enum for SDK level, i will update this. It will just equals()

Reply:
That sounds right. We should be able to reuse the same implementation model now then?

Reply:
it will be update to
`            assertTrue(expectedMark.getState().toString().equals(actualMark.getState().toString()));
`